### DISCUSSION TITLE
Tufte样式，纠结了 | /cn/2011/04/tuftes-book-style/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#702] [Comments]
https://yihui.org/cn/2011/04/tuftes-book-style/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
可能是因为有一次LyX崩溃的缘故，过段时间我会结合读者反馈修正这些问题。R语言介绍还要重写，那一章是几年前写的，不成系统，但也不能删掉，因为要照顾初学者。

pdfcrop裁不了从tikz生成的PDF，必须用它看了bounding box的信息在人工指定裁的范围，这就是麻烦所在。

> ——原帖发布于 2011-04-15 07:32:57

[a comment by an automated account is not shown]

**Reply by pumc-zhou** [reply]
可以在rstudio中的创建页面选择tufte的中文模板

**Comment by pumc-zhou** [comment]
rmarkdown中的tufte样式是很好的，不过找了一圈好像没有找到汉化成功的中文tufte模版，现在只能曲线救国了。

**Reply by yihui** [reply]
多年之后我已经实现了全面曲线救国，也就是纯 HTML 实现精美 PDF 的理想。示例在 https://yihui.org/litedown/ 4.2 小节。在默认网页样式下按 p 键即可自动排版为 PDF 样式，tufte 样式只是其中一个小特例。

**Reply by pumc-zhou** [reply]
谢谢您的回复与创作，您的回复让我看到了新的可能，非常精美的网页，我会继续学习。

**Comment by yangxx0405** [comment]
有tex的汉化版吗
